test/test_baseline_trainers.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from glob import glob
+
+import pytest
+
+try:
+    from habitat_baselines.run import run_exp
+    from habitat_baselines.common.base_trainer import BaseRLTrainer
+    from habitat_baselines.config.default import get_config
+
+    baseline_installed = True
+except ImportError:
+    baseline_installed = False
+
+
+@pytest.mark.skipif(
+    not baseline_installed, reason="baseline sub-module not installed"
+)
+@pytest.mark.parametrize(
```

Comment:
This is amazing way to cover all trainers smoke test with one parameterized test. Thank you for implementing it.